vips/vip-408/bsctestnet.ts

```diff
@@ -0,0 +1,144 @@
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MocksUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MocksUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, parseUnits("10000", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VsUSDe_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_Ethena,
+            parseUnits("0.9", 18),
+            parseUnits("0.92", 18),
+            parseUnits("10000", 18),
+            sepolia.VTREASURY,
+            parseUnits("50000000", 18),
+            parseUnits("0", 18),
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [MockUSDC, parseUnits("10000", 6), sepolia.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MockUSDC,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MockUSDC,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, parseUnits("10000", 6)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VUSDC_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VUSDC_Ethena,
+            parseUnits("0", 18),
+            parseUnits("0", 18),
+            parseUnits("10000", 6),
+            sepolia.VTREASURY,
+            parseUnits("50000000", 6),
+            parseUnits("46000000", 6),
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VsUSDe_Ethena,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.010", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: VUSDC_Ethena,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.020", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: COMPTROLLER_ETHENA,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[VsUSDe_Ethena, VUSDC_Ethena], [2, 7], true],
```

Comment:
We cannot pause the "enter market" action for USDC

Reply:
[1866d37](https://github.com/VenusProtocol/vips/pull/440/commits/1866d377e77845c38ab5502842a9fee5915bfbb9)